util/netevent.h

```diff
@@ -216,6 +233,18 @@ struct comm_point {
 	struct sldns_buffer* http_temp;
 	/** http stored content in buffer */
 	size_t http_stored;
+	/* -------- HTTP/2 ------- */
+	/** http2 session */
+	struct http2_session* h2_session;
+	/** set to 1 if h2 is negatiated using alpn */
```

Comment:
negotiated.  I seem to recall if's in callbacks on alpn_h2 that seem to assume it is always used for http/2 connections.